mkdocs.yml

```diff
@@ -44,6 +44,33 @@ theme:
 plugins:
   - search
 
+nav:
```

Comment:
Tímto si nejsem jist. Už jsem taky koketoval s řazením a nelíbí se mi nutnost vypisovat strukturu všech souborů 😕. Hledal jsem jestli nemají nějaký plugin nebo něco na to řazení ale vypadá to že bohužel. Nebo berem že se do toho handbooku tolik sahat nebude tak je to jen dočasný diskomfort si vždy přidat path k novému návodu?

Reply:
Bohužial sa to zatiaľ javí ako jediné riešenie, asi s tým budeme musieť nejak žiť. Zas to nie je až taký pain udržiavať.